db/re/item_db_equip.yml

```diff
@@ -98736,6 +99023,39 @@ Body:
     Locations:
       Costume_Garment: true
     ArmorLevel: 1
+  - Id: 20991
+    AegisName: aegis_20991
+    Name: Fairy of Eden
+    Type: Armor
+    Weight: 400
+    Defense: 40
+    Slots: 1
+    Locations:
+      Garment: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus2 bSubClass,Class_Normal,5;
+      bonus2 bSubClass,Class_Boss,5;
+      bonus2 bSubRace,RC_Player_Human,5;
+      bonus2 bSubRace,RC_Player_Doram,5;
+      bonus bAspdRate,.@r;
+      bonus bMatkRate,2*.@r;
+      if (.@r>=6) {
+         bonus bAllStats,10;
+         bonus bNoKnockback;
+      }
+      if (.@r>=8) {
+         skill "MG_ENERGYCOAT",1;
+         if (getskilllv("MG_ENERGYCOAT") == 1) {
```

Comment:
You mean `autobonus3 "{ bonus bDef,300; bonus bMdef,15; }",1000,300000,"MG_ENERGYCOAT";` ?

Reply:
Translation of reference 2
https://forums.warpportal.com/index.php?/topic/211129-jro-costama-ragcan-translation-thread/page-5#entry2994628

Reply:
`If upgrade level is 8 or higher, enebles Energy Coat Lv1, if Energy Coat is Lv1, Def + 300, Mdef + 15.`

I mean since the item give the skill Energy Coat the bonus `if Energy Coat is Lv1, Def + 300, Mdef + 15.` is always given
unless they mean they give the bonus only if the user has already Energy Coat, then they give the skill
```
      if (.@r>=8) {
         if (getskilllv("MG_ENERGYCOAT") == 1) {
            bonus bDef,300;
            bonus bMdef,15;
         }
         skill "MG_ENERGYCOAT",1;
      }
```
You may let it as you did, it's only a thought

Reply:
sorry i misunderstood

Reply:
Thanks !